simpleml/metrics/classification.py

```diff
@@ -86,6 +86,23 @@ def validate_predictions(predictions):
 
 
 class BinaryClassificationMetric(ClassificationMetric):
+    @property
+    def labels(self):
+        labels = self.model.get_labels(dataset_split=self.dataset_split)
```

Comment:
nit - can we change this to `super(BinaryClassificationMetric, self).labels` with a comment to make it clear that this is extending parent label retrieval with a validation step for binary values. Unlikely, but potential, room for divergence otherwise